tests/e2e/specs/MessageFlows.spec.js

```diff
@@ -99,17 +125,14 @@ describe('Message Flows', () => {
 
     getNumberOfLinks().should('equal', 0);
 
-    connectNodesWithFlow('message-flow-button', poolPosition, poolPosition);
```

Comment:
Connecting to itself isn't an invalid target?

Reply:
It is, but leaving it here for some reason caused this test to fail on CI. So I've moved it into the test above, "Cannot connect to itself", where I think it made more sense anyway:

```js
[poolPosition, startEventPosition, taskPosition].forEach(position => {
  connectNodesWithFlow('message-flow-button', position, position);
});
```